socorro/external/boto/crash_data.py

```diff
@@ -59,6 +59,9 @@ def get(self, **kwargs):
         if not params.uuid:
             raise MissingArgumentError('uuid')
 
+        if not ooid.is_crash_id_valid(params.uuid):
+            raise BadArgumentError('uuid')
+
```

Comment:
This looks good. It's definitely worth adding a test.

Looks like the tests for the `RawCrash` api are here:

https://github.com/mozilla-services/socorro/blob/b486727bf42f52327f3140ba4bcd35288df3868a/webapp-django/crashstats/crashstats/tests/test_models.py#L319

That mocks out `.get()` on the `SimplifiedCrashData`. Since you're testing what happens if there are bad crash ids, I think you can skip the mock and verify it raises an exception.

See if that works. If not, something similar should work.